com.unity.multiplayer.mlapi/Runtime/Profiling/ITransportProfilerData.cs

```diff
@@ -1,6 +1,6 @@
 using System.Collections.Generic;
 
-namespace Unity.Multiplayer.Netcode.Profiling
+namespace Unity.Multiplayer.Netcode
```

Comment:
we might still want to keep network collections behind a namespace like `Unity.Multiplayer.Netcode.Profiling`, what would you guys think? @becksebenius-unity @Rosme 

also, should any of these types be `public` at all? I though they all could be `internal` types, no?

Reply:
okay, I see that `NetworkManager` implements `IProfilableTransportProvider` interface which has a property returning `ITransportProfilerData` type. I guess, if we were to keep these two `public` and make everything `internal`, that'd work fine and we'd still keep them under `Unity.Multiplayer.Netcode` namespace — how that sounds to you guys? 👀

Reply:
`ITransportProfilerData` will ultimately become deprecated, it's not something that we're using for Profiler V2. In general I'd say we should make as much of this internal as possible, although this was public because it needs to be used in other assemblies

Reply:
I'm aligned with @becksebenius-unity